WordPressAuthenticator/GoogleSignIn/JWToken.swift

```diff
@@ -0,0 +1,47 @@
+/// Represents a JSON Web Token (JWT)
+///
+/// See https://jwt.io/introduction
+struct JWToken {
+    let encodedValue: String
```

Comment:
A minor suggestion: Swift uses the term `rawValue` in lots of its APIs, I feel like we can use the same variable name here?

Reply:
Sounds good. 

Addressed in 99ac474f5cb293168ff256d43fdc850ced46db53